src/ImageSharp/PixelFormats/PixelBlenders/PorterDuffFunctions.cs

```diff
@@ -189,38 +203,58 @@ public static Vector4 Atop(Vector4 dst, Vector4 src, Vector4 blend)
             return color;
         }
 
+        /// <summary>
+        /// Returns the result of the "In" compositing equation.
+        /// </summary>
+        /// <param name="destination">The destination vector.</param>
+        /// <param name="source">The source vector.</param>
+        /// <param name="blend">The amount to blend. Range 0..1</param>
+        /// <returns>The <see cref="Vector4"/>.</returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static Vector4 In(Vector4 dst, Vector4 src, Vector4 blend)
+        public static Vector4 In(Vector4 destination, Vector4 source, Vector4 blend)
         {
-            float alpha = dst.W * src.W;
+            // TODO: blend is not used?
+            float alpha = destination.W * source.W;
 
-            Vector4 color = src * alpha;                    // premultiply
+            Vector4 color = source * alpha;                    // premultiply
             color /= MathF.Max(alpha, Constants.Epsilon);   // unpremultiply
             color.W = alpha;
 
             return color;
         }
 
+        /// <summary>
+        /// Returns the result of the "Out" compositing equation.
+        /// </summary>
+        /// <param name="destination">The destination vector.</param>
+        /// <param name="source">The source vector.</param>
+        /// <returns>The <see cref="Vector4"/>.</returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static Vector4 Out(Vector4 dst, Vector4 src)
+        public static Vector4 Out(Vector4 destination, Vector4 source)
```

Comment:
`Out` and `Xor` are called by several generated methods but no blend vector is created or used for each method which means identical results . I would expect a blend vector to be generated and passed.